src/reducers/nodes.js

```diff
@@ -43,6 +44,34 @@ function nodeReducer(nodeState = {}, action) {
       });
     }
 
+    case ADD_NODE_METADATA: {
+      const {id, data} = action.data;
+      console.log('id', id)
```

Comment:
leftover console.log

Reply:
aargh oops my bad, thanks for the great spot!